test/test_main.cpp

```diff
@@ -0,0 +1,73 @@
+/**
+ * @file
+ * @brief     Test file
+ * @author    Jeroen van Hattem
+ * @license   MIT
+ */
+#define CATCH_CONFIG_MAIN // This tells Catch to provide a main() - only do this in one cpp file
+#include "../src/coordinate3d.hpp"
+#include "../src/robot_arm.hpp"
+
+#include "catch.hpp"
+
+RobotArm uarmSwiftPro;
+char text[100];
+
+TEST_CASE("Append char* with another char*") {
```

Comment:
For the small test cases, doxygen does not provide much information. But as test cases start to get really big, it is advised to write doxygen for them as well.